projects/storefrontapp-e2e-cypress/cypress/helpers/vendor/cpq/quote-download.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as authentication from '../../auth-forms';
+
+export const READ_QUOTE = '@READ_QUOTE';
+export const READ_VENDOR_QUOTE = '@READ_VENDOR_QUOTE';
+export const DOWNLOAD_ATTACHMENT = '@DOWNLOAD_ATTACHMENT';
+const SHOP_NAME = Cypress.env('BASE_SITE'); //Powertools-spa
+
+/**
+ * Uses a cx-login-form to login a user.
+ *
+ * @param email Email for the login
+ * @param password Password for the login
+ * @param name Name of the user
+ */
+export function login(email: string, password: string, name: string): void {
+  cy.get('cx-login [role="link"]')
+    .click()
+    .then(() => {
+      cy.get('cx-login-form').should('be.visible');
+    });
+  authentication.login(email, password);
+  cy.get('.cx-login-greet').should('contain', name);
+  cy.get('cx-login').should('not.contain', 'Sign In');
+}
+
+/**
+ * Uses a cx-login-form to log out a user.
+ */
+export function logout(): void {
```

Comment:
There is `logout(... )` method line 224 (see projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-quote.ts).
Please, use this one.